### PR TITLE
Add UI for calculator

### PR DESCRIPTION
**Pull request by shaaibu7**
- [ ] Implement design using JSX and CSS
- [ ] Use functional components

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+function Calculator() {
+  return (
```
Comment (danmainah) [inline src/components/Calculator.js:4]: - [ ] @shaaibu7 kindly check the design of your calculator at some widths esp small screen it's broken, below is a screenshot of how it currently looks 
image

Comment (danmainah) [reply src/components/Calculator.js:4]: - [ ] @shaaibu7 kindly ensure you have a child component for your Calculator.js component, this is a project requirement
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @shaaibu7 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used


# Required Changes ♻️
- [ ] @shaaibu7 kindly delete all the boilerplate from Create React App (text, images, styles) in your public folder.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢 

Hi @shaaibu7,

Good job working on the changes highlighted by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Lionter checks passed 👍🏽
✅ Created `Claculator.js` component as required 👍🏽
✅ Your app runs fine with no error 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**